tests/drivers/display/display_read_write/boards/da1469x_dk_pro.overlay

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include <zephyr/dt-bindings/gpio/gpio.h>
+ #include <zephyr/dt-bindings/display/panel.h>
+
+ / {
+    chosen {
+        zephyr,display = &display;
+    };
+
+    // aliases {
```

Comment:
Need to drop these commented blocks in the overlay

Reply:
OK. As I explained before, the reason I've commented out this part is that the MEMC driver has yet to be merged into main.